mysql/README.md

```diff
@@ -1,59 +1,131 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog agent's MySQL check sends many database metrics to Datadog, including metrics for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And [many more](https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv). You can also invent your own metrics using custom queries.
 
-1. Create a ```datadog``` user with replication rights in your MySQL server:
+The MySQL check sends one service check: whether or not the Datadog agent can connect to MySQL.
 
-        sudo mysql -e "CREATE USER 'datadog'@'localhost' IDENTIFIED BY '<UNIQUEPASSWORD>';"
-        sudo mysql -e "GRANT REPLICATION CLIENT ON *.* TO 'datadog'@'localhost' WITH MAX_USER_CONNECTIONS 5;"
-2. Verify that the above commands worked by running:
+It does not send anything to your events stream.
```

Comment:
Maybe "The MySQL check does not emit any events"? Or maybe another sentence to clarify that you can still receive events, such as monitor notifications from this check, even though it doesn't emit any events of its own?